lib/travis/build/appliances/services.rb

```diff
@@ -24,6 +24,15 @@ def apply
               end
             end
 
+            if services.delete('mysql')
+              sh.if "$(grep -E '^socket = /run/mysql-5.6' #{HOME_DIR}/.my.cnf)" do
+                sh.cmd "sudo service mysql-5.6 start", assert: false, echo: true, timing: true
```

Comment:
The service name change should be announced and documented for people who change the configuration. According to GitHub search, there are +/- 250 public repos who have `sudo service mysql` in the build config. Can the service be renamed/aliased?